src/main/webapp/js/calendar.js

```diff
@@ -80,54 +80,54 @@ function initCalendar() {
   calendar.setDate(new Date('2000-01-02T00:00:00'));
 }
 
- /**
-   * Adds a course to the calendar.
-   * @param {Object} course The JSON Object for the course.
-   */
-  async function addCourse(course) {
-    if (course == null) {
-      return;
-    }
-    const response = await fetch(
-        `/api/sections?course_id=${encodeURIComponent(course.course_id)}`);
-    const json = await response.json();
-
-    // For now, just choose the first section out of the available ones
-    if (json.sections == null) {
-      return;
-    }
-    const firstSection = json.sections[0];
-    if (firstSection.meetings == null) {
-      return;
-    }
-    const firstMeetingInfo = firstSection.meetings[0];
-    const meetingDays = firstMeetingInfo.days;
-
-    const startTime = firstMeetingInfo.start_time;
-    const endTime = firstMeetingInfo.end_time;
-
-    if (meetingDays.includes('Su')) {
-      addCourseToCalendar(course, startTime, endTime, enumDays.DATE_SUNDAY);
-    }
-    if (meetingDays.includes('M')) {
-      addCourseToCalendar(course, startTime, endTime, enumDays.DATE_MONDAY);
-    }
-    if (meetingDays.includes('Tu')) {
-      addCourseToCalendar(course, startTime, endTime, enumDays.DATE_TUESDAY);
-    }
-    if (meetingDays.includes('W')) {
-      addCourseToCalendar(course, startTime, endTime, enumDays.DATE_WEDNESDAY);
-    }
-    if (meetingDays.includes('Th')) {
-      addCourseToCalendar(course, startTime, endTime, enumDays.DATE_THURSDAY);
-    }
-    if (meetingDays.includes('F')) {
-      addCourseToCalendar(course, startTime, endTime, enumDays.DATE_FRIDAY);
-    }
-    if (meetingDays.includes('Sa')) {
-      addCourseToCalendar(course, startTime, endTime, enumDays.DATE_SATURDAY);
-    }
+/**
```

Comment:
If possible, remove these js files from the PR. I know that it's annoying because the auto formatter picks up on these, but william said that we should create a separate PR that fixes these files so that PRs like this stay cleaner.